### PR TITLE
Added a call for contribution in the pong tutorial

### PR DESCRIPTION
**Pull request by Moxinilian**
This should both:

- Encourage contributing on doc (and hopefully then on code)
- Increase traffic to our subreddit as it will now be mentioned in the book

### CODE REVIEW

File: book/src/pong-tutorial/contribution.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Call for contribution
+
+And... that's where the pong chapter ends.
+Amethyst is still in development and writing good tutorials is hard, so the guide is not complete yet.
+We are sorry this might feel a bit disappointing.
+
+Fortunately, the pong example is complete, and you can find the entire code with balls, score and music on the example pages available [here](https://www.amethyst.rs/doc). You can get additional help by leaving a post on [our subreddit](https://www.reddit.com/r/Amethyst) or on our Discord server.
```
Comment (torkleyy) [inline book/src/pong-tutorial/contribution.md]: Please link that, too.

File: book/src/pong-tutorial/contribution.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+We hate to have an incomplete tutorial as much as you do, and you can help us complete it!
+As stated above, the code is ready, tested and working. The only thing we need now is good English.
+Even if you have limited experience with the engine, if you think you know how to write a tutorial, you can reach us out and we will help you do it by giving you details on how things work and how to structure it.
```
Comment (torkleyy) [inline book/src/pong-tutorial/contribution.md]: Actually, "good enough" English skills and good knowledge of the engine would be preferable IMO. Probably both would help.

File: book/src/pong-tutorial/contribution.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Call for contribution
+
+And... that's where the pong chapter ends.
+Amethyst is still in development and writing good tutorials is hard, so the guide is not complete yet.
+We are sorry this might feel a bit disappointing.
+
+Fortunately, the pong example is complete, and you can find the entire code with balls, score and music on the example pages available [here](https://www.amethyst.rs/doc). You can get additional help by leaving a post on [our subreddit](https://www.reddit.com/r/Amethyst) or on our [Discord server](https://discord.gg/GnP5Whs).
+
+We hate to have an incomplete tutorial as much as you do, and you can help us complete it!
+As stated above, the code is ready, tested and working. The only thing we need now is good enough English.
```
Comment (jojolepro) [inline book/src/pong-tutorial/contribution.md]: The only thing we need now is good enough English

Sounds a bit weird imo

Comment (Moxinilian) [reply book/src/pong-tutorial/contribution.md]: Does it?

Comment (Moxinilian) [reply book/src/pong-tutorial/contribution.md]: "The only thing we are in need" sounds weirder to me

Comment (torkleyy) [reply book/src/pong-tutorial/contribution.md]: Honestly, @Moxinilian it sounded a bit strange to me, too. I  think you can leave that sentence out. We have enough people with that skill, the missing resources are either time or engine knowledge.

**Comment by Moxinilian** [comment]
bors r=torkleyy,jojolepro

**Pull request merged**